### PR TITLE
test_multiprocessing.py fix incorrect use of numpy.random.randint

### PR DESCRIPTION
**Pull request by ahundt**
There was a recent pull request that appeared to fix the problem, but actually it just ran into a different failure case. This actually fixes it. :-)

### CODE REVIEW

**Comment by fchollet** [comment]
Can you explain in more detail:

- Unit tests were previously passing, so what was the problem?
- Why would we be throwing a naked Exception? In general, we should not do that
- If we are, why do we need to except 2 Exception types?

**Comment by ahundt** [comment]
#6765 4a6f06f changed Exception => ValueError, running the tests locally occasionally it will threw Exception also, but I'm not very familiar with it. 

I think I investigated briefly and it looked like it was due to multiple workers. One code path would throw Exception in an underlying python library, the other throws ValueError. Test success/failure could be a race condition.

**Comment by ahundt** [comment]
The problem was incorrect usage of `numpy.random.randint(low,high,...)`, which throws an `Exception` when low>high. Updated PR with a correct fix.

File: tests/test_multiprocessing.py [hunk 1]
```diff
@@ -270,27 +270,26 @@ def custom_generator():
...
+            yield (np.random.randint(1, 256, (2, 50)),
+                   np.random.randint(1, 256, (2, 50)))
         raise RuntimeError
 
     model = Sequential()
     model.add(Dense(1, input_shape=(2, )))
     model.compile(loss='mse', optimizer='adadelta')
 
-    with pytest.raises(ValueError):
+    with pytest.raises((ValueError)):
```
Comment (fchollet) [inline tests/test_multiprocessing.py]: Parentheses are no longer necessary, in this case?

**Comment by fchollet** [comment]
LGTM

**Pull request merged**